.gitmodules

```diff
@@ -52,3 +52,6 @@
 [submodule "software/example-user-syscall"]
 	path = software/example-user-syscall
 	url = https://github.com/sifive/example-user-syscall.git
+[submodule "software/coremark"]
+	path = software/coremark
+	url = https://github.com/sifive/coremark.git
```

Comment:
This should probably point at sifive/benchmark-coremark.git

Reply:
Updated.